pkg/daemon/ceph/client/pool_test.go

```diff
@@ -572,7 +573,10 @@ func testCreatePoolWithReplicasPerFailureDomain(t *testing.T, failureDomain, cru
 			assert.Equal(t, args[4], "size")
 			poolSize, err := strconv.Atoi(args[5])
 			assert.NoError(t, err)
-			assert.Equal(t, uint(poolSize), poolSpec.Replicated.Size)
+			if poolSize < math.MaxUint {
```

Comment:
Looks like a build error:
```
Error: vet: pkg/daemon/ceph/client/pool_test.go:576:18: 
math.MaxUint (untyped int constant 18446744073709551615) overflows int
```

Reply:
right. I've seen that one locally with `make go.vet`. comparison has an implicit cast.

Still thinking how to fix this

Reply:
fixed